### ISSUE TITLE
Review EFO cross-references for obsolete terms

### ISSUE THREAD

**Issue by sbello** [DiseaseOntology/HumanDiseaseOntology#1285]
The EFO cross-reference for Parkinson's disease (DOID:14330), EFO:0002508, has been obsoleted and replaced with a MONDO class. 

Same thing for:
DOID:0050741

DOID:11782 - EFO:0004222 has been replaced by an HP term

May be worth setting up/running a QC to look for obsolete cross-references.

**Comment by allenbaron** [comment]
Running `make update_efo` will now automatically remove obsolete EFO xrefs and produce a report of the changes. The report lists only the xref changes and does not include the reason for the obsolescence(s).

I ran this and removed all the xrefs to obsoleted EFO terms that were identifiable to this point. I also used the SPARQL query in the PR to view the reasons for obsolescence and all of them were replaced by Mondo or HP terms. I suspect there will be need to run this on occasion as EFO replaces more terms with Mondo or HP terms.

If you see other EFO terms in the future that have been obsoleted let us. Otherwise, I'll probably run this once or year or so.

Thanks again, Sue.
